scalyr_agent/agent_main.py

```diff
@@ -1671,9 +1671,10 @@ def __log_copy_manager_stats(self, overall_stats):
         stats = overall_stats
 
         log.info(
-            "copy_manager_status total_scan_iterations=%ld total_read_time=%lf total_compression_time=%lf total_waiting_time=%lf total_blocking_response_time=%lf "
+            "copy_manager_status workers_number=%ld total_scan_iterations=%ld total_read_time=%lf total_compression_time=%lf total_waiting_time=%lf total_blocking_response_time=%lf "
```

Comment:
Just thinking out loud - I don't think this should break any PowerQueries or similar since I don't think any of that values ordering.

If it does / for just in case, we could add it at the end.

Reply:
I am not sure, but it looks like each graph does its own line parsing, so we shouldn't worry about that.